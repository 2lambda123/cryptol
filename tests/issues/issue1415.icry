:set timeQuiet=on
:time 0x1 + 0x2
:t it
