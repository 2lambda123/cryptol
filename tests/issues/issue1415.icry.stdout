Loading module Cryptol
it : {avgTime : Float 11 53, avgCpuTime : Float 11 53, avgCycles : Integer}
